client/data/multi-currency/selectors.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+/**
+ * Retrieves the multiCurrency state from the wp.data store if the state
+ * has been initialized, otherwise returns an empty state.
+ *
+ * @param {Object} state Current wp.data state.
+ *
+ * @return {Object} The multiCurrency state.
+ */
+const getMultiCurrencyState = ( state ) => {
+	if ( ! state ) {
+		return {};
+	}
+
+	return state.multiCurrency || {};
+};
+
+export const getAvailableCurrencies = ( state ) => {
+	const available = getMultiCurrencyState( state ).available || {};
```

Comment:
It looks to me like `available` and `enabled` would be arrays that contain currencies, is that right? If so, we should update their default values to be `[]`.

Reply:
They are likely going to be arrays, yes. Thanks for catching that, updated in https://github.com/Automattic/woocommerce-payments/pull/1856/commits/491b258ba28a0cde79b1d8f9055b32605591fe5c